docker-compose.yml

```diff
@@ -1,8 +1,135 @@
 ---
-version: "3.7"
+version: '3'
 services:
-  mongodb:
-    image: ${MONGODB_IMAGE:-percona/percona-server-mongodb:4.2}
-    container_name: mnogo_exporter_mongodb
-    ports:
-      - 127.0.0.1:27017:27017
+  standalone:
+    network_mode: host
```

Comment:
why do all instances here have `network_mode: host`?

Reply:
⚠️ Yeah, that will not work Mac, and we expect other developers to work on this exporter on Macs.